taca/server_status/cli.py

```diff
@@ -13,13 +13,38 @@ def server_status():
 		raise RuntimeError("Configuration missing required entries: server_status")
 
 
-
 # server status subcommands
 @server_status.command()
-@click.option('--credentials', type=click.Path(exists=True), default=os.path.join(os.path.dirname(__file__), 'gdocs_credentials.json'),
+@click.option('--gdocs', is_flag=True, help="Update the google docs")
+@click.option('--statusdb', is_flag=True, help="Update the statusdb")
+@click.option('--credentials', type=click.Path(exists=True), default=os.path.join(os.environ.get('HOME'), '.taca', 'gdocs_credentials.json'),
 				 help='Path to google credentials file')
-def disk_space(credentials):
-	""" Checks the available space on all the disks
+def nases(credentials, gdocs, statusdb):
+	""" Checks the available space on all the nases
+	"""
+	disk_space = status.get_nases_disk_space()
+	if gdocs:
+		status.update_google_docs(disk_space, credentials)
+	if statusdb:
+		status.update_status_db(disk_space)
+
+#  must be run on uppmax, as no passwordless ssh to uppmax servers
```

Comment:
Well in reality there is a password less ssh connection, under my user name in bioinfo.seq user. But it will likely change with the new cluster so better to not use this now